packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx

```diff
@@ -135,7 +135,13 @@ export const DataSourceHttpSettings: React.FC<HttpSettingsProps> = (props) => {
         <h3 className="page-heading">HTTP</h3>
         <div className="gf-form-group">
           <div className="gf-form">
-            <FormField label="URL" labelWidth={13} tooltip={urlTooltip} inputEl={urlInput} />
+            <FormField
+              aria-label="Datasource HTTP settings url"
```

Comment:
This is never applied to the element since you're passing an `inputEl`.
Despite that, this PR shows me a useful way of using selectors for the e2e tests!